pycbc/workflow/minifollowups.py

```diff
@@ -652,3 +655,10 @@ def create_noop_node():
     exe.add_pfn(pfn)
     node = wdax.Node(exe)
     return node
+
+def add_wiki_row(outfile, cols):
```

Comment:
I think this function makes more sense in the script itself, given that it isn't likely to be used by other codes and is pretty specific. Is the idea to have a generic 'write wiki format' function?